COPYRIGHT

```diff
@@ -1,34 +1,1073 @@
+ .
+  If you develop a new program, and you want it to be of the greatest
+ possible use to the public, the best way to achieve this is to make
+ it free software which everyone can redistribute and change under
+ these terms.
+ .
+  To do so, attach the following notices to the program.  It is safest
+ to attach them to the start of each source file to most effectively
+ state the exclusion of warranty; and each file should have at least
+ the "copyright" line and a pointer to where the full notice is found.
+ .
+    <one line to give the program's name and a brief idea of what it does.>
+    Copyright (C) <year>  <name of author>
+ .
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+ .
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+ .
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ .
+ Also add information on how to contact you by electronic and paper mail.
+ .
+  If the program does terminal interaction, make it output a short
+ notice like this when it starts in an interactive mode:
+ .
+    <program>  Copyright (C) <year>  <name of author>
+    This program comes with ABSOLUTELY NO WARRANTY; for details type `show w'.
+    This is free software, and you are welcome to redistribute it
+    under certain conditions; type `show c' for details.
+ .
+ The hypothetical commands `show w' and `show c' should show the appropriate
+ parts of the General Public License.  Of course, your program's commands
+ might be different; for a GUI interface, you would use an "about box".
+ .
+  You should also get your employer (if you work as a programmer) or
+ school, if any, to sign a "copyright disclaimer" for the program, if
+ necessary.  For more information on this, and how to apply and follow
+ the GNU GPL, see <http://www.gnu.org/licenses/>.
+ .
+  The GNU General Public License does not permit incorporating your
+  program into proprietary programs.  If your program is a subroutine
+  library, you may consider it more useful to permit linking
+  proprietary applications with the library.  If this is what you want
+  to do, use the GNU Lesser General Public License instead of this
+  License.  But first, please read
+  <http://www.gnu.org/philosophy/why-not-lgpl.html>.
 
-The EFF wordlist (file ``wordlsts/wordlist_en_eff.txt``) is licensed
-by the Electronic Frontier Foundation under the `Creative Commons
-CC-BY 3.0 US` license (see
-https://creativecommons.org/licenses/by/3.0/us/).
-
-"Diceware" is a trademark of A G Reinhold.
```

Comment:
i believe the trademark notice may be missing from the final file.

Reply:
It is. I wasn't sure if a trademark notice belongs in the copyright file, and there's no way to represent it in the machine readable version as far as I know. I'm also not sure what to say about the trademark status, since we're waiting to hear back from Reinhold.

Reply:
well, if you are going to remove the trademark notice, i would make that at least a separate commit or maybe even a whole different PR since it is a policy change. reading the title of this PR, it seems to say it only transforms the file, but this removes a trademark notice, so i would make that more obvious somehow.

but i would keep the notice: you can insert arbitrary notes in the text, for example with a [Comment](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#comment-field) field in the header...

Reply:
Yeah, you're right. I've added the trademark comment back.